### ISSUE TITLE
Board Of Directors Single Person Post 

### ISSUE THREAD

**Issue by poopsplat** [firebelly/cfs#18]
It seemed to me that the individual people pages were only going to be used for `staff` type person posts, but I saw on production that a bio was added to a `board-of-directors` post, and the individual page is broken because there was no image added to that post (https://chicagofreedomschool.org/about-us/people/timmy-chau/) — I suggest either simply adding a photo to that user, or adjusting the markup for the Our People page to only show a link to the full bio page if it is a staff member. @Markj89 any thoughts on this?

**Comment by Markj89** [comment]
Could I add a conditional if Image is present?

**Comment by poopsplat** [comment]
Yeah! As in, add a conditional for displaying the link to the full bio on the Our People page only if that person has an image set? If that's what you mean, then I say yeah, makes sense — go for it!

**Comment by Markj89** [comment]
Can you send me screenshots of what you're seeing because in my local, I just see a 404 on timmy-chau page.

**Comment by poopsplat** [comment]
@Markj89 — I see exactly what I see here: https://chicagofreedomschool.org/about-us/people/timmy-chau/ — you may need to refresh the permalinks if you're switching from the press branch by going into Settings > Permalinks and just hitting save. Also make sure you run gulp anew to make sure you get the right styles.
